docs/pages/blog/introducing-the-row-grouping-feature.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Introducing the Row Grouping feature'
+description: After more than a year of development on the DataGrid, the first premium feature is live.
+date: 2022-01-20T00:00:00.000Z
+authors: ['alexfauquette']
+tags: ['News']
```

Comment:
```suggestion
tags: ['News', 'MUI X']
```
cc @oliviertassinari

Reply:
I don't see the use case for the tags "News" all blog posts are news, from what I understand, but a 💯 for the product label with MUI X.

Reply:
News in the sense that it's of value for the user, not that it is a piece of new information only. Combining product with post category (MUI X + News) is useful because we could also have other content categories within the same product, such as MUI X + How-to guides or MUI X + Case studies, to name a few examples.

Reply:
I think that I need one thing to understand the label: what's the pain that we solve? What I assume is that it's all about being able to **filter**. Assuming that we write blog post to be valuable for user/readers and using the above definition ("that it's of value for the user/reader"), I conclude that it's all "News". 

If we could answer the following, it would be perfect:

1. Who will use this tag to filter the blog posts? 
2. What's the rule to know if "'News" should be applied or not? 

For "Company", I envision employees and the people who care about the development of the company, like candidates. For "MUI Core" and "MUI X", I get it the audience only using one of the products.

Reply:
To clarify, I think that _News_ should be posts that have a direct impact on how the product is used. That's why it makes sense for me to have posts introducing any new feature or changes to the current product experience tagged as News. That's what I mean with "it's of value for the user". A blog post announcing a team member is just not as relevant news for developers out there using MUI Material, for example. It is news, yes, but not one that produces changes to the product's capabilities or developer experience.

> Why https://mui.com/blog/2021/ doesn't have it?

I'd say that because it's not introducing anything new, it's a recap post. 

> Who (audience type) will use the "News" tag to filter the blog posts?

I'd ask a different question: _What type of content should be tagged as News?_ for which I'd suggest: the ones that change the product some way.

Reply:
@danilo-leal Oh, so it's about the product then? How about we rename "News" to "Product", as it's done by

- https://www.docker.com/blog/

<img width="586" alt="Screenshot 2022-01-25 at 12 51 33" src="https://user-images.githubusercontent.com/3165635/150972010-05348529-f519-41a5-85e5-463eca41f8ac.png">

- https://stripe.com/blog
<img width="357" alt="Screenshot 2022-01-25 at 12 51 39" src="https://user-images.githubusercontent.com/3165635/150972014-fd1d2847-730e-49ea-b1fe-91f06d2be34c.png">

- https://www.hashicorp.com/blog

<img width="484" alt="Screenshot 2022-01-25 at 12 52 01" src="https://user-images.githubusercontent.com/3165635/150972065-46f34a18-64b9-490c-9734-7a2668fbdb4b.png">

- https://strapi.io/blog

<img width="407" alt="Screenshot 2022-01-25 at 12 52 40" src="https://user-images.githubusercontent.com/3165635/150972165-74ce0828-54cd-4d76-8a66-1fe118e4ae05.png">


I could see the usage of News in [MongoDB](https://www.mongodb.com/blog/channel/news) and [AWS](https://aws.amazon.com/blogs/?awsf.blog-master-category=category%23news). Personally, it makes no sense to me. I don't understand 1. (who would use it) nor 2. (how a blog author know if he should add the label or not)

Reply:
> How about we rename "News" to "Product"

I wouldn't, especially given that we added MUI Core and X, which are already products. Stripe, for example, uses tags depending on discipline (engineering, product, corporate). The other ones use products as we use MUI Core and MUI X, kinda. 

> Who (audience type) will use the "News" tag to filter the blog posts?

I guess it's anyone interested in seeing release posts (and not notes) of relevant changes. It could be really anyone, developers trying to catch up, designers trying to understand, etc. 

> What's the rule to know if "'News" should be applied on a blog post or not? Why https://mui.com/blog/2021/ doesn't have it? We share new information, like the roadmap that reader can care about.

It's because the 2021 post doesn't have an announcement or introduction of anything new, it's a recap post. It sure has maybe new info about roadmap and stuff like that but it doesn't introduce anything that will change developers' experience today. The PR we're using for having this conversation, for example, does. It introduces a new Data Grid feature that developers can use right away. Jun's v5.3 post does the same. For me, the rule is: does it introduce any change to the product? Such one that developers can use right away and check the post for more understanding? If yes, then it is tagged as News.

Reply:
I propose we move this thread to https://mui-org.notion.site/Improve-blog-posts-infrac-bcae2b15493f4daba0758adf499f7793 as the discussion that I started is off-topic regarding the main purpose of the PR. Currently it's a draft exportation, but I could move it to a GitHub issue if preferred.